internal/storage/fs/object/blob/blob_fs_test.go

```diff
@@ -52,7 +52,7 @@ func Test_FS(t *testing.T) {
 		}
 		require.NoError(t, bucket.Close())
 
-		azfs, err := NewFS(logger, "file", dir)
+		azfs, err := NewFS(ctx, StrUrl(fileblob.Scheme, dir), dir, "data/")
 		require.NoError(t, err)
 
 		// running test
```

Comment:
I sadly can't comment on the line in question because it is not a part of this PR.
However, down on 80 the expected map now needs to contain the keys:
```go
expected := map[string]string{
    "data/one": "onedata",
    "data/two": "twodata",
}
```